pkg/ocm/storage/received/ocm.go

```diff
@@ -284,7 +284,7 @@ func convertStatToResourceInfo(ref *provider.Reference, f fs.FileInfo, share *oc
 		Id:       id,
 		MimeType: mime.Detect(f.IsDir(), f.Name()),
 		Path:     name,
-		Name:     name,
+		Name:     filepath.Base(name),
```

Comment:
Isn't this a bug in the webdav module? I guess we should fix it there. Shouldn't we?

Reply:
Moved to the gowebdav